data/assets/scene/solarsystem/planets/earth/noaa-sos/atmosphere/carbontracker_2000_2100-fixed_scale.asset

```diff
@@ -0,0 +1,110 @@
+local globeIdentifier = asset.require("../../earth").Earth.Identifier
+
+
+local Name = "CarbonTracker: Fixed Scale"
+local Identifier = "noaa-sos-atmosphere-carbontracker_2000_2100-fixed_scale"
+local Description = [[ "NOAA encourages science that adds benefit to society and the 
+environment. CarbonTracker does both." said retired Navy Vice Admiral Conrad 
+Lautenbacher, Ph.D., former undersecretary of commerce for oceans and atmosphere and NOAA 
+administrator. CarbonTracker is a system to keep track of carbon dioxide uptake and 
+release at the Earth's surface over time. It was developed by the Carbon Cycle Greenhouse 
+Gases group at NOAA's Earth System Research Laboratory. As a scientific tool, 
+CarbonTracker has helped improve the understanding of carbon uptake and release from the 
+land and oceans, and how those sources and sinks are responding to the changing climate, 
+increasing levels of atmospheric CO2 (the CO2 fertilization effect), and human management 
+of land and oceans. CarbonTracker relies on the long-term monitoring of atmospheric CO2 
+performed by the NOAA Global Monitoring Division and international partners.  ]]
+local URL = "https://sos.noaa.gov/catalog/datasets/carbon-tracker-fixed-scale/"
+
+
+local syncedDirectory = asset.syncedResource({
+  Name = Name,
+  Type = "HttpSynchronization",
+  Identifier = Identifier,
+  Version = 1
+})
+
+local fieldDestination = syncedDirectory .. "field/"
```

Comment:
I'd disagree since the meaning behind them is different (see https://gcc.godbolt.org/z/3P8xsPvv5)
the version without the "/" at the end is pointing *at* the directory, the one with the trailing / points *into* the directory.  In all of these asset cases we want to refer to the folder itself, rather than treating it as the first argument to a path substitution